### PR TITLE
Reigndocs

### PR DESCRIPTION
**Pull request by reign-git**

### CODE REVIEW

File: docs/synapse/userguides/storm_ref_lift.ipynb [hunk 1]
```diff
@@ -623,6 +624,170 @@
...
+  {
+   "cell_type": "raw",
+   "metadata": {
+    "hideCode": false
+   },
+   "source": [
+    "Safe Lifts\n",
+    "----------\n",
+    "\n",
+    "\"Safe\" lifts, also known as \"try\" lifts, are lifts performed in a safe and error-free manner. Safe lifts refer to lifts that perform :ref:`data-type` validation before attempting to lift a query node, and fails safely if an invalid Type is encountered. Specifically, it prevents a Cortex from throwing a runtime execution error, and terminating query execution when Type validation is unsuccessful for a given property value.\n",
```
Comment (vEpiphyte) [inline docs/synapse/userguides/storm_ref_lift.ipynb:649]: so there isn't confusion - the type validation is always done when trying to do lifts. the try operators simply suppress errors in type normalizing; so we're not doing a extra check, we just don't let the error tear down the runtime

File: docs/synapse/userguides/storm_ref_lift.ipynb [hunk 2]
```diff
@@ -623,6 +624,170 @@
...
+    "\n",
+    "When lifting nodes by property value using the equals (``=``) comparator, Type validation is not performed on the supplied property value prior to query execution. If a query is executed with an invalid Type, or a Type that does not correspond to the supplied form, the Cortex will throw a ``BadTypeValu`` error, and terminate the query as shown below."
```
Comment (vEpiphyte) [inline docs/synapse/userguides/storm_ref_lift.ipynb:651]: still confusing here - we always do that validation, we just allow those errors to tear down the runtime

File: docs/synapse/userguides/storm_ref_lift.ipynb [hunk 3]
```diff
@@ -719,11 +719,11 @@
   {
    "cell_type": "raw",
    "metadata": {
-    "hideCode": true,
+    "hideCode": false,
     "hideOutput": false
    },
    "source": [
-    "- Attempt to safely lift the DNS nodes whose ``inet:dns:a:ipv4`` secondary property value equals ``'192.168.0.100'``. Notice that an error is not generated, despite an invalid IPv4 address ``'192.168.0.1000'`` being entered:"
+    "- Try to lift the DNS nodes whose ``inet:dns:a:ipv4`` secondary property value equals ``'192.168.0.100'``. Notice that an error message is displayed.despite an invalid IPv4 address ``'192.168.0.1000'`` being entered:"
```
Comment (vEpiphyte) [inline docs/synapse/userguides/storm_ref_lift.ipynb:726]: shouldn't this display no error message?

**Review by vEpiphyte — APPROVED** [review]

**Comment by reign-git** [comment]
Try lift section added to user doc lift section.

**Pull request merged**